Makefile.client

```diff
@@ -146,7 +146,10 @@ grpc: $(LIBGRPCDIR)/libgrpc_indicator
 
 $(LIBGRPCDIR)/libgrpc_indicator:
 	mkdir -p $(LIBGRPCDIR)
-	git clone -b $(shell curl -L https://grpc.io/release) \
+	# Build gRPc v1.19.0 since it is a known good build. Change to
```

Comment:
Remove this file from your commit. Just rebase to master and you will pick up the change.